tribits/core/package_arch/TribitsAdjustPackageEnables.cmake

```diff
@@ -1480,9 +1480,9 @@ macro(tribits_adjust_package_enables)
   endforeach()
 
   if (${PROJECT_NAME}_DUMP_PACKAGE_DEPENDENCIES)
-    message("\nDumping direct dependencies for each SE package ...")
-    foreach(TRIBITS_PACKAGE ${${PROJECT_NAME}_SE_PACKAGES})
-      tribits_print_direct_package_dependencies_lists(${TRIBITS_PACKAGE})
+    message("\nDumping direct dependencies for each package ...")
+    foreach(tribitsPkg  IN  LISTS  ${PROJECT_NAME}_DEFINED_TPLS  ${PROJECT_NAME}_SE_PACKAGES)
```

Comment:
I find that I have trouble seeing the spaces between arguments in some CMake code because there are no comas or other delimitators like `some_long_text_string and_does_this_keep_going`.  I find it easier to see the gap when I use two spaces in these cases to give  `some_long_text_string  and_does_this_keep_going`.

So I have gravitated to using two spaces to separate arguments that have no other delimitators.  But I find it is easy to see like `...some_long_variable_name} some_other_long_argument` or `some_long_other_argument ${some_long_variable_name...`.  The presence of  `}` or `${` fixes the problem for me and I don't need the extra space to see the gap.

And, to be consistent, I have gravitated gravitated to using two spaces for all of these cases.

And the case for `IN LISTS` with `foreach()` is very confusing to me because I always mix this up with `IN_LIST` with `if()`.  I keep wanting to write `foreach( ... IN_LISTS ...)` instead of `foreach( ... IN LISTS ...)` (which is a very annoying bug to track down).